doc/build-openbsd.md

```diff
@@ -50,7 +50,7 @@ export BDB_PREFIX="$PWD/db4"
 
 ### Building Bitcoin Core
 
-**Important**: use `gmake`, not `make`. The non-GNU `make` will exit with a horrible error.
+**Important**: Use `gmake` (the non-GNU `make` will exit with an error).
 
 Preparation:
 ```bash
```

Comment:
nit: since you replaced `bash` with `shell` in `build-netbsd.md`, if you would be updating the patch, maybe `s/bash/shell/` also here in `build-openbsd.md`. But the patch is good to go as is IMO since its purpose is to deal with `MAKE=gmake` issue.

Reply:
Thanks, I've addressed this.